packages/forma-36-website/src/pages/guidelines/accessibility/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Accessibility'
+---
+
+# Accessibility
+
+Our aim is to design and build inclusive features and applications which are usable for users of all abilities.
+
+## Introduction
+
+When we solve problems and create new features we don't want to be biased by our individual abilities but be emphatic to users who are limited in interacting with the provided user interface. Impairments are not always permanent but can be temporary or situational too, so we want to provide a variety of ways how to interact with our application. The WCAG provides guidance on how to include users of all abilities and qualifies a level of accessibility by providing success criteria. Contentful aims to meet the WCAG 2.1 Level AA success criteria.
+
+## Assistive Technologies
+
+There are several forms of limitations a user can face when interacting with a user interface, they can be roughly categorised as follows:
+
+- Auditory
+- Cognitive
+- Neurological
+- Physical
+- Visual
+
+All major operating systems provide a certain amount of assistive technologies which enable alternative ways of communicating content and functionality especially for users with auditory and visual impairments. This includes tools like contrast switchers, screen readers or braille display support. It is important to create content which is accessible and to provide technical implementations to enable those technologies. The following lists should provide simple guides on what to look out for and prevent common accessibility failures:
+
+### Text Alternatives
+
+- All non-text content should provide alternative text (e.g. `alt` attribute for images, `label` or `describedby` aria attribute)
+
+### Adaptable
+
+- Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. E.g. Provide labels for form components or use the aria `describedby` attribute to create relationship between content.
```

Comment:
Use parentheses around the example for consistency:
...available in **text (e.g.** provide labels for form components...